src/main/scala/com/itechart/ny_accidents/utils/DateUtils.scala

```diff
@@ -26,14 +33,41 @@ object DateUtils {
     Try(LocalDateTime.parse(dateStr, dateFormatter)).toOption
   }
 
+  def parseSqlDate(dateString: String, dateFormatter: DateTimeFormatter): Option[SqlDate] = {
+    Try(new SqlDate(LocalDate.parse(dateString, dateFormatter)
+      .atStartOfDay().toInstant(ZoneOffset.UTC).toEpochMilli)) match {
+      case Success(value) => Some(value)
+      case Failure(exception) =>
+        println(exception)
```

Comment:
Never print to console

Reply:
@abelyaev-itechart 
fixed
e18dbc8a6616d0d03a051a8ef80c3b59a11017ac